### PR TITLE
#106 add admin flash messages

### PR DESCRIPTION
**Pull request by denys-medynskyi**
I have troubles with test javascript manipulated flash message. Can someone give me a hand ? @sveredyuk

### CODE REVIEW

File: app/controllers/admin/events_controller.rb [hunk 1]
```diff
@@ -9,11 +9,21 @@ def new
     def create
       @event = Event.new(event_params)
 
-      event.save ? default_redirect : render(:new)
+      if event.save
+        flash[:success] = t('events.notifications.create.success')
```
Comment (denys-medynskyi) [inline app/controllers/admin/events_controller.rb]: also what do you think about this namespacing of notifications ?

Comment (sveredyuk) [reply app/controllers/admin/events_controller.rb]: @denysmedynskyi i think we need DRY.
Implement some method like `flash_success(:events)`

**Comment by sveredyuk** [comment]
@denysmedynskyi don't test js in admin area. it's useless...

File: app/controllers/admin/events_controller.rb (hunk 1 above)
Comment (denys-medynskyi) [reply app/controllers/admin/events_controller.rb]: cool

File: app/controllers/admin/base_controller.rb [hunk 2]
```diff
@@ -3,5 +3,15 @@ class BaseController < ApplicationController
     layout 'admin'
 
     before_action :authenticate_user!
+    after_action :prepare_unobtrusive_flash
+
+    def flash_success(resource_name)
+      flash[:success] = t("#{resource_name.to_s}.notifications.#{params[:action]}.success")
```
Comment (denys-medynskyi) [inline app/controllers/admin/base_controller.rb:9]: @sveredyuk flash methods like this ?

Comment (sveredyuk) [reply app/controllers/admin/base_controller.rb:9]: Yes

**Comment by denys-medynskyi** [comment]
resolve #106

**Pull request merged**